test/test_helpers/fake_fs.rb

```diff
@@ -18,3 +18,11 @@ def teardown
     end
   end
 end
+
+module FakeFS
+  class File
+    def self.binwrite(*args)
```

Comment:
The `fakefs` gem doesn't have this method, but I can open a PR and fix it there. This will just unblock us for now

Reply:
My 2 cents is that since `context.binread` and `.binwrite` are such trivial methods, and it's probably not worth unit testing if it's this much trouble due to the nature of them.  I think we just need to update the rest of the unit tests to use a fake `Context` that doesn't actually read from FS.  WDYT?

Reply:
I agree with this ^

Reply:
That makes sense to me! 👍